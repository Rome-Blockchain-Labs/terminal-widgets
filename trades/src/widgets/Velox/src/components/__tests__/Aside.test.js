import { shallow } from 'enzyme';

import { Aside } from '../Aside';

describe('Aside component render', () => {
  it('Aside snapshot added', () => {
    const component = shallow(<Aside />);
    expect(component).toMatchSnapshot();
  });
});
