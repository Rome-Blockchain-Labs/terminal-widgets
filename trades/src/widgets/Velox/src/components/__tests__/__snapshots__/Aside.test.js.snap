// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside component render Aside snapshot added 1`] = `
<div
  className="container"
>
  <div
    className="aside"
  >
    <h2
      className="aside__title section-title shadow-box"
    >
      Current Bot Operations
    </h2>
    <div
      className="aside__info shadow-box"
    >
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          Token Pair: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          DEX Used: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          Token Amount: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          ETH Amount: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          BUY IF: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
           SELL IF: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          Gas Cap: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <p
        className="aside__info-row"
      >
        <span
          className="aside__info-title"
        >
          Dynamic Threshold: 
        </span>
        <span
          className="aside__info-value"
        >
          XXXX
        </span>
      </p>
      <VeloxButton
        className="--wide"
        prefix="aside"
      >
        cancel
      </VeloxButton>
    </div>
  </div>
</div>
`;
